### DISCUSSION TITLE
Experiencing lag despite using a better GPU

### DISCUSSION THREAD

**Discussion by Br1Ms7one** [hyprwm/Hyprland#8571] [Q&A] [answered]
Dealing with lag on my setup, even with an Nvidia GPU working alongside my Intel integrated GPU. What could be the fix?

**Comment by xaicat** [comment, marked as answer]
This sounds like a GPU configuration issue, especially in multi-GPU setups. To fix it:

1. Check your Hyprland config to ensure the correct GPU is prioritized for external displays.
2. This might be helpful for you: -https://wiki.hyprland.org/Configuring/Multi-GPU/

Also, make sure your Nvidia drivers are up-to-date. Lag can occur if the external monitor defaults to the integrated GPU instead of the Nvidia card.

**Comment by Br1Ms7one** [comment]
Thanks
